### DISCUSSION TITLE
Use a collective taskbar

### DISCUSSION THREAD

**Discussion by kindusmithkindu** [linuxmint/discussions#383] [Ideas]
Ubuntu gnome, ubuntu kylin, zorin os, windows, macos, and kde all use a collective taskbar to put network settings, Bluetooth, volume, keyboard, brightness, etc. in the same tray. Will Linuxmint adopt it?

**Comment by Secret-chest** [comment]
Do they? Maybe they're trying to copy phones. Just because everyone does the same, doesn't mean Lm can't do better.

**Comment by rcalixte** [comment]
This doesn't have to be an all or nothing idea. Mint/Cinnamon doesn't need to adopt it but it doesn't stop anyone from creating an Applet with that desired functionality. Just like Cinnamenu or the alternative window list applets, there is nothing inhibiting an alternative from being developed and the published for the community via Spices. All it needs it someone with the time and dedication to see it through development and then maintain it.
